build.gradle

```diff
@@ -5,6 +5,10 @@ repositories {
 }
 
 dependencies {
+    compile 'com.fasterxml.jackson.core:jackson-core:2.6.3'
+    compile 'com.fasterxml.jackson.core:jackson-databind:2.6.3'
+    compile 'com.fasterxml.jackson.core:jackson-annotations:2.6.3'
```

Comment:
In fact it was enough if we used `jackson-databind` 😄 Both `jackson-core` and `jackson-annotations` are dependencies of `jackson-databind` so they are included automatically.